src/runtime/test/thrift_message_parser_test.cpp

```diff
@@ -215,7 +222,7 @@ TEST_F(thrift_message_parser_test, get_message_on_receive_incomplete_second_fiel
         int read_next = 0;
         message_reader reader(64);
         data = std::string("THFT") + std::string(i, ' ');
-        mock_reader_read_data(reader, data);
+        mock_reader_read_data(reader, data, 1);
```

Comment:
Here we can set `mock_reader_read_data` with default message_num to 1.

Reply:
done.